.github/ISSUE_TEMPLATE/02-feature_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Feature Request
+description: If you have a suggestion
+labels: [suggestion]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please use a concise and distinct title for the issue.
+  - type: textarea
+    attributes:
+      label: Pitch
+      description: Describe your idea for a feature. Make sure it has not already been suggested/implemented/turned down before.
```

Comment:
```suggestion
      description: Describe your idea for a feature. Please check our [issue tracker](https://github.com/Retrospring/retrospring/issues?q=is%3Aissue+label%3Asuggestion) to make sure it has not already been suggested/implemented/turned down before.
```

Reply:
I'm not sure if the description of fields supports Markdown, considering there is a specific `markdown` field type for text info. We'd need to verify that first.